src/libraries/libraries-packages.proj

```diff
@@ -16,7 +16,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(PkgDir)*\*.proj" Exclude="$(PkgDir)test\*" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)*\pkg\**\*.pkgproj" Condition="('$(BuildAllConfigurations)' == 'true' or '$(DotNetBuildFromSource)' == 'true') And '$(BuildAllOOBPackages)' == 'true'" />
+    <PkgProjectsPath Include="$(MSBuildThisFileDirectory)*\pkg\**\*.pkgproj" Condition="('$(BuildAllConfigurations)' == 'true' or '$(DotNetBuildFromSource)' == 'true') And '$(BuildAllOOBPackages)' == 'true'" />
```

Comment:
Can we instead just pack every CSProj?  If a CSProj has a PKGProj it will pack that.  If a CSProj doesn't have a PKGProj then it will use the default Pack target which can be disabled with IsPackable (or enabled rather on the projects that want to pack).